packages/toolpad-app/next.config.mjs

```diff
@@ -167,6 +167,6 @@ export default /** @type {import('next').NextConfig} */ ({
         source: '/:path*',
         headers: securityHeaders,
       },
-    ]
```

Comment:
@apedroferreira Any idea on how this passed the `prettier` CI check up until now?

Reply:
not really, maybe if you saved it before it's not running for files with the `.mjs` extension?
i probably just saved the file and prettier added this

Reply:
Did `prettier` get updated recently? CI only checks for changed files I believe. Perhaps we should run `prettier:all` in CI?